blackbelt/dependencies.py

```diff
@@ -325,7 +325,7 @@ def traverse_dependencies(dependencies, path):
         for dependency_name in dependencies:
             dependency = dependencies[dependency_name]
 
-            if dependency_name == name and dependency['version'] == version:
+            if dependency_name == name and 'version' in dependency and dependency['version'] == version:
```

Comment:
```python
and dependency.get('version') == version
```

Reply:
🤔 how can you have a dependency without a version?

Reply:
`npm ls` on a locally installed app (`bb dep check .`) with missing peer deps generate one that doesn't contain the version information.